pkg/services/publicdashboards/api/api.go

```diff
@@ -174,10 +177,12 @@ func (api *Api) QueryPublicDashboard(c *models.ReqContext) response.Response {
 // util to help us unpack dashboard and publicdashboard errors or use default http code and message
 // we should look to do some future refactoring of these errors as publicdashboard err is the same as a dashboarderr, just defined in a
 // different package.
-func handleDashboardErr(defaultCode int, defaultMsg string, err error) response.Response {
+func (api *Api) handleError(code int, message string, err error) response.Response {
```

Comment:
My only concern with this change is that `code` will not always be used inside this func. Someone using this function could think that the code sent to it will be the code returned by the Response?

Reply:
mmm I see. Maybe we should change them back to the previous names?